hieradata/common.yaml

```diff
@@ -389,9 +389,11 @@ govuk::apps::frontend::nagios_memory_warning: 1200
 govuk::apps::frontend::nagios_memory_critical: 1400
 govuk::apps::frontend::redis_host: "%{hiera('sidekiq_host')}"
 govuk::apps::frontend::redis_port: "%{hiera('sidekiq_port')}"
+govuk::apps::government-frontend::unicorn_worker_processes: "3"
```

Comment:
should this one be `govuk::apps::frontend::unicorn_worker_processes` ?

Reply:
I really was feeling funny yesterday :-/